### DISCUSSION TITLE
why is the vue package on version 0.8, while react and solid are on 1.0.1

### DISCUSSION THREAD

**Discussion by stefanprobst** [chakra-ui/ark#1685] [General]
curious what the status is with `@ark-ui/vue`?

**Comment by cschroeter** [comment]
Needs a bit more love

**Reply by stefanprobst** [reply]
maybe the docs could add a short note about the current state of the vue package? because it seems that the examples in the docs don't currently work with vue? (e.g. for https://ark-ui.com/docs/components/select, which complains about missing `Select.Control`, `Select.ValueText`, etc.).

**Reply by cschroeter** [reply]
@stefanprobst we will iterate over each Vue component and also file a 1.0 release shortly.

**Reply by enescagl** [reply]
Is there any progress for this? I am currently installing it from git repo using [gitpkg](https://gitpkg.vercel.app/). But npm package would be preferable.

**Reply by Omikorin** [reply]
@enescagl we just released v0.9
